setup/instance/inputs.go

```diff
@@ -188,8 +190,10 @@ func RequiredStringPrompter(m ModuleInput, current interface{}) (interface{}, er
 		for i := 0; i < 3; i++ {
 			fmt.Printf("\tInput: ")
 
-			var input string
-			fmt.Scanln(&input)
+			// Allow spaces for input
+			reader := bufio.NewReader(os.Stdin)
```

Comment:
where/why do we want spaces for inputs? This impl is swallowing errors.

Reply:
I want the ability to add file paths with spaces.

Reply:
From golang's docs for [ReadLine](https://golang.org/pkg/bufio/#Reader.ReadLine), they suggest using a [Scanner](https://golang.org/pkg/bufio/#Scanner):

```go
scanner := bufio.NewScanner(os.Stdin)
scanner.Scan()
if err := scanner.Err(); err != nil {
    return err
}

input := scanner.Text()
```